sdk/go/auto/workspace.go

```diff
@@ -109,6 +109,22 @@ type Workspace interface {
 	StackOutputs(context.Context, string) (OutputMap, error)
 }
 
+type RemoveStackOption interface {
+	getRemoveStackFlag() string
+}
+
+func RemoveStackForce() RemoveStackOption {
+	return &removeStackFlagOption{flag: "--force"}
```

Comment:
This implementation is too specific to the fact that the fact that the automation api uses the CLI. Rather than flag, let's create a specific force option. It might look something along the lines of:

```go
type removeStackOptions {
  Force bool
}

type RemoveStackOptions interface {
	applyRemoveStackOption(*removeStackOptions)
}

type removeStackOption func(*removeStackOptions)

func (o removeStackOption) applyRemoveStackOption(opts *removeStackOptions) {
	o(opts)
}

func Force() RemoveStackOptions {
	return removeStackOption(func(wo *removeStackOptions) {
		wo.Force = true
	})
}
```

In addition, let's extract this option code into a separate package in `sdk/go/auto/optremove` to keep the auto namespace clean (there are likely other commands that `Force` could apply to). This is what we have done for preview, update, refresh, and destroy options: https://github.com/pulumi/pulumi/blob/c3dc2d54abf85da811de0fbaa5e98f22b0f054c8/sdk/go/auto/optup/optup.go#L17

Reply:
Done

Reply:
Looks like we should be able to clean up the original options and remove them from `workspace.go`. Would you mind doing that?

Reply:
Yeah sorry about that, forgot to clean that up